frontend/src/component/changeRequest/ChangeRequest/Changes/Change/SegmentChangeDetails.tsx

```diff
@@ -48,7 +48,14 @@ export const SegmentChangeDetails: VFC<{
     const { segment: currentSegment } = useSegment(change.payload.id);
 
     return (
-        <SegmentContainer>
+        <SegmentContainer
+            sx={theme => ({
+                borderColor: change.conflict
+                    ? theme.palette.warning.border
+                    : theme.palette.divider,
+                borderTopColor: theme.palette.divider,
+            })}
+        >
```

Comment:
I'd rather do this directly in the styled component, but I'm not sure if that's possible with a dynamic variable. Is there a way to make styled components accept more than `theme`?

Reply:
Try to find `shouldForwardProp` in our frontend code and you'll find a recipe

Reply:
One example
```
const StyledValue = styled('div', {
    shouldForwardProp: prop => prop !== 'color',
})(({ color }) => ({
    textAlign: 'left',
    width: '100%',
    color: color,
}));
```

Reply:
Thanks 😄